pkgs/top-level/python-packages.nix

```diff
@@ -6548,7 +6540,11 @@ in {
 
   poster3 = callPackage ../development/python-modules/poster3 { };
 
-  postorius = callPackage ../servers/mail/mailman/postorius.nix { };
+  postorius = throw "Please use pkgs.mailmanPackages.postorius";
+  hyperkitty = throw "Please use pkgs.mailmanPackages.hyperkitty";
+  mailman = throw "Please use pkgs.mailman";
+  mailman-hyperkitty = throw "Please use pkgs.mailmanPackages.mailman-hyperkitty";
+  mailman-web = throw "Please use pkgs.mailman-web";
```

Comment:
These should go in `python-aliases.nix`.

Reply:
Done.